code/__HELPERS/_logging.dm

```diff
@@ -167,6 +168,10 @@
 	SEND_TEXT(world.log, text)
 
 
+/proc/log_paper(text)
+	WRITE_LOG(GLOB.world_paper_log, "PAPER: [text]")
```

Comment:
why are hrefs behind a config but logging paper is not?

paper seems the lesser of the two?
(although log file will be significantly smaller).

Reply:
not every one is behind a config. the href already existed it just did nothing.